<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>931</width>
    <height>727</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                   stop: 0 #E0E0E0, stop: 1 #FFFFFF);
                        
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="1" column="1">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="minimumSize">
       <size>
        <width>50</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba( 0, 0, 0, 30% );
border:1px solid ;
border-radius:10px


 </string>
      </property>
      <property name="title">
       <string/>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">border:&quot;none&quot; ;                       
                        
background-color: &quot;transparent&quot;;
</string>
         </property>
         <property name="text">
          <string>Repet </string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="pushButton_p">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>50</height>
          </size>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 white, stop: 1 grey);
border-style: solid;
border-color: black;
border-width: 1px;
border-radius: 10px;</string>
         </property>
         <property name="text">
          <string>Perceptron</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="pushButton_h">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>90</width>
           <height>50</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>30</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="mouseTracking">
          <bool>false</bool>
         </property>
         <property name="tabletTracking">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 white, stop: 1 grey);
border-style: solid;
border-color: black;
border-width: 1px;
border-radius: 10px;</string>
         </property>
         <property name="text">
          <string>Heb</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLineEdit" name="repet">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>0</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color:rgb(255, 255, 227);
                                        border: 1px solid gray;
                                        border-radius: 10px;
                                        padding: 0 8px;
                                        selection-background-color: darkgray;
                                        font-size: 16px;
                                       </string>
         </property>
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QPushButton" name="pushButton_clear">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>60</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>50</height>
          </size>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 white, stop: 1 grey);
border-style: solid;
border-color: black;
border-width: 1px;
border-radius: 10px;</string>
         </property>
         <property name="text">
          <string>Clear</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QGroupBox" name="groupBox">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>MS Shell Dlg 2</family>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba( 0, 0, 0, 30% );
border:1px solid ;
border-radius:10px


 </string>
      </property>
      <property name="title">
       <string/>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="3" column="0">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="3" column="1">
        <widget class="QPushButton" name="Ok">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>70</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="focusPolicy">
          <enum>Qt::ClickFocus</enum>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 white, stop: 1 grey);
border-style: outset;
border-color: black;
border-width: 1px;
border-radius: 10px;</string>
         </property>
         <property name="text">
          <string>Ok</string>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="1" colspan="3">
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="4" rowspan="3">
          <widget class="Line" name="line">
           <property name="minimumSize">
            <size>
             <width>12</width>
             <height>0</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </widget>
         </item>
         <item row="0" column="5">
          <widget class="QLineEdit" name="lineEdit_A11">
           <property name="styleSheet">
            <string notr="true">background-color:rgb(255, 255, 227);
                                        border: 1px solid gray;
                                        border-radius: 10px;
                                        padding: 0 8px;
                                        selection-background-color: darkgray;
                                        font-size: 16px;
                                       </string>
           </property>
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item row="1" column="8">
          <widget class="QLabel" name="label_ave_A">
           <property name="styleSheet">
            <string notr="true">border:&quot;none&quot; ;                       
                        
background-color: &quot;transparent&quot;;
</string>
           </property>
           <property name="text">
            <string>Ave A</string>
           </property>
          </widget>
         </item>
         <item row="2" column="6">
          <widget class="QLineEdit" name="lineEdit_A22">
           <property name="styleSheet">
            <string notr="true">background-color:rgb(255, 255, 227);
                                        border: 1px solid gray;
                                        border-radius: 10px;
                                        padding: 0 8px;
                                        selection-background-color: darkgray;
                                        font-size: 16px;
                                       </string>
           </property>
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item row="1" column="5" colspan="2">
          <widget class="Line" name="line_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="2" column="5">
          <widget class="QLineEdit" name="lineEdit_A21">
           <property name="styleSheet">
            <string notr="true">background-color:rgb(255, 255, 227);
                                        border: 1px solid gray;
                                        border-radius: 10px;
                                        padding: 0 8px;
                                        selection-background-color: darkgray;
                                        font-size: 16px;
                                       </string>
           </property>
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item row="0" column="7" rowspan="3">
          <widget class="Line" name="line_2">
           <property name="minimumSize">
            <size>
             <width>12</width>
             <height>0</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QLabel" name="label_Cov_A">
           <property name="styleSheet">
            <string notr="true">border:&quot;none&quot; ;                       
background-color: &quot;transparent&quot;;
</string>
           </property>
           <property name="text">
            <string>Cov A</string>
           </property>
          </widget>
         </item>
         <item row="0" column="6">
          <widget class="QLineEdit" name="lineEdit_A12">
           <property name="styleSheet">
            <string notr="true">background-color:rgb(255, 255, 227);
                                        border: 1px solid gray;
                                        border-radius: 10px;
                                        padding: 0 8px;
                                        selection-background-color: darkgray;
                                        font-size: 16px;
                                       </string>
           </property>
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item row="2" column="9">
          <widget class="QLineEdit" name="lineEdit_Ave_A2">
           <property name="styleSheet">
            <string notr="true">background-color:rgb(255, 255, 227);
                                        border: 1px solid gray;
                                        border-radius: 10px;
                                        padding: 0 8px;
                                        selection-background-color: darkgray;
                                        font-size: 16px;
                                       </string>
           </property>
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item row="0" column="9">
          <widget class="QLineEdit" name="lineEdit_Ave_A1">
           <property name="styleSheet">
            <string notr="true">background-color:rgb(255, 255, 227);
                                        border: 1px solid gray;
                                        border-radius: 10px;
                                        padding: 0 8px;
                                        selection-background-color: darkgray;
                                        font-size: 16px;
                                       </string>
           </property>
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="3" column="2">
        <widget class="QLabel" name="label_number">
         <property name="styleSheet">
          <string notr="true">border:&quot;none&quot; ;                       
                        
background-color: &quot;transparent&quot;;
</string>
         </property>
         <property name="text">
          <string>number</string>
         </property>
        </widget>
       </item>
       <item row="3" column="3">
        <widget class="QLineEdit" name="number">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color:rgb(255, 255, 227);
                                        border: 1px solid gray;
                                        border-radius: 10px;
                                        padding: 0 8px;
                                        selection-background-color: darkgray;
                                        font-size: 16px;
                                       </string>
         </property>
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" colspan="3">
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="4" rowspan="3">
          <widget class="Line" name="line_4">
           <property name="minimumSize">
            <size>
             <width>12</width>
             <height>0</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </widget>
         </item>
         <item row="0" column="5">
          <widget class="QLineEdit" name="lineEdit_B11">
           <property name="styleSheet">
            <string notr="true">background-color:rgb(255, 255, 227);
                                        border: 1px solid gray;
                                        border-radius: 10px;
                                        padding: 0 8px;
                                        selection-background-color: darkgray;
                                        font-size: 16px;
                                       </string>
           </property>
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item row="1" column="8">
          <widget class="QLabel" name="label_Ave_B">
           <property name="styleSheet">
            <string notr="true">border:&quot;none&quot; ;                       
                        
background-color: &quot;transparent&quot;;
</string>
           </property>
           <property name="text">
            <string>Ave B</string>
           </property>
          </widget>
         </item>
         <item row="2" column="6">
          <widget class="QLineEdit" name="lineEdit_B22">
           <property name="styleSheet">
            <string notr="true">background-color:rgb(255, 255, 227);
                                        border: 1px solid gray;
                                        border-radius: 10px;
                                        padding: 0 8px;
                                        selection-background-color: darkgray;
                                        font-size: 16px;
                                       </string>
           </property>
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item row="1" column="5" colspan="2">
          <widget class="Line" name="line_5">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="2" column="5">
          <widget class="QLineEdit" name="lineEdit_B21">
           <property name="styleSheet">
            <string notr="true">background-color:rgb(255, 255, 227);
                                        border: 1px solid gray;
                                        border-radius: 10px;
                                        padding: 0 8px;
                                        selection-background-color: darkgray;
                                        font-size: 16px;
                                       </string>
           </property>
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item row="0" column="7" rowspan="3">
          <widget class="Line" name="line_6">
           <property name="minimumSize">
            <size>
             <width>12</width>
             <height>0</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QLabel" name="label_Cov_B">
           <property name="styleSheet">
            <string notr="true">border:&quot;none&quot; ;                       
                        
background-color: &quot;transparent&quot;;
</string>
           </property>
           <property name="text">
            <string>Cov B</string>
           </property>
          </widget>
         </item>
         <item row="0" column="6">
          <widget class="QLineEdit" name="lineEdit_B12">
           <property name="styleSheet">
            <string notr="true">background-color:rgb(255, 255, 227);
                                        border: 1px solid gray;
                                        border-radius: 10px;
                                        padding: 0 8px;
                                        selection-background-color: darkgray;
                                        font-size: 16px;
                                       </string>
           </property>
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item row="2" column="9">
          <widget class="QLineEdit" name="lineEdit_Ave_B2">
           <property name="styleSheet">
            <string notr="true">background-color:rgb(255, 255, 227);
                                        border: 1px solid gray;
                                        border-radius: 10px;
                                        padding: 0 8px;
                                        selection-background-color: darkgray;
                                        font-size: 16px;
                                       </string>
           </property>
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item row="0" column="9">
          <widget class="QLineEdit" name="lineEdit_Ave_B1">
           <property name="styleSheet">
            <string notr="true">background-color:rgb(255, 255, 227);
                                        border: 1px solid gray;
                                        border-radius: 10px;
                                        padding: 0 8px;
                                        selection-background-color: darkgray;
                                        font-size: 16px;
                                       </string>
           </property>
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="0" rowspan="5">
     <widget class="MplWidget" name="MplWidget" native="true"/>
    </item>
    <item row="2" column="1" rowspan="3">
     <widget class="QGroupBox" name="groupBox_3">
      <property name="minimumSize">
       <size>
        <width>300</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba( 0, 0, 0, 30% );
border:1px solid ;
border-radius:10px


 </string>
      </property>
      <property name="title">
       <string/>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
      <layout class="QGridLayout" name="gridLayout_6">
       <item row="0" column="0">
        <widget class="QTextBrowser" name="textBrowser">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 white, stop: 1 grey);
border-style: solid;
border-color: black;
border-width: 1px;
border-radius: 10px;</string>
         </property>
         <property name="html">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;1.First, use the mean and  covariance to produce your samples in the desired number.&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;2. in the classification section, a bad class can bedone with to methods, perceptron and heb.&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MplWidget</class>
   <extends>QWidget</extends>
   <header>mplwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
